Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/DownloadConfigMapper.kt

```diff
@@ -2,20 +2,93 @@ package de.rki.coronawarnapp.appconfig.mapping
 
 import dagger.Reusable
 import de.rki.coronawarnapp.appconfig.KeyDownloadConfig
+import de.rki.coronawarnapp.diagnosiskeys.server.LocationCode
 import de.rki.coronawarnapp.server.protocols.internal.AppConfig
-import de.rki.coronawarnapp.server.protocols.internal.KeyDownloadParameters
+import de.rki.coronawarnapp.server.protocols.internal.KeyDownloadParameters.KeyDownloadParametersAndroid
+import org.joda.time.Duration
+import org.joda.time.LocalDate
+import org.joda.time.LocalTime
+import org.joda.time.format.DateTimeFormat
+import timber.log.Timber
 import javax.inject.Inject
 
 @Reusable
 class DownloadConfigMapper @Inject constructor() : KeyDownloadConfig.Mapper {
```

Comment:
I'm not so fond of unnecessary shortcuts in naming... why not calling this `KeyDownloadParametersMapper` according to the ProtoBuf structure?